### ISSUE TITLE
As a DevSecOps engineer, I want to build a pipeline for Parabol with RethinkDB side car based on the approved UBI selection from #4400

### ISSUE THREAD

**Issue by adaniels-parabol** [ParabolInc/parabol#4402]
As a DevSecOps engineer, I want to build a pipeline for Parabol with RethinkDB side car based on the approved UBI selection from #4400 

_Need the build process and working deployment before starting security scanning. This is to validate application is working as intended in UBI images with proper configuration_

AC:

- [x] Build Parabol application pipeline

  - [x] Create Dockerfile for Parabol application with all required commands and configurations

  - [x] Build stage for Parabol application

  - [x] Deployment stage for Parabol application

  - [x] Validate build and deployment stages for Parabol application

- [x] Build RethinkDB pipeline

  - [x] Create Dockerfile for RethinkDB with all required commands and configurations

  - [x] Build stage for RethinkDB

  - [x] Seed RethinkDB with minimal dataset for testing

  - [x] Deployment stage for RethinkDB

  - [x] Validate build and deployment stages for RethinkDB

- [x] Include RethinkDB as sidecar in Parabol application deployment

- [x] Validate functional Parabol application

**Comment by adaniels-parabol** [comment]
TL/DR: we have a sandbox to validate security changes to Parabol, this includes a functioning deployment. We get to inherit base os security via the RHEL UBI from Ironbank

I stood up an Ironbank deployment on our GCP K8s instance. I needed a place to test builds and operational deployments of Parabol as we get results back from the hardening pipeline over on Ironbank. This means, I needed a place using hardened images for postgres, redis, and rethinkdb (we are supporting the rethinkdb hardened image), and parabol.

Thus, https://ironbank.proving-ground.parabol.co/ was born :baby: (be sure to use incognito mode as any current cookies for Parabol will cause an error on this instance, also use "create account" with un/pw as sso and google auth aren't configured, only internal auth, so you'll need to make an account

This deployment of Parabol is using the hardened universal base images form Ironbank (already approved for use) which is Redhat Linux (e.g. RHEL). All I've done is make modifications to run Parabol. This will give us a place to validate security updates to pass hardening approval at Ironbank, but also allow us to run on the same hardened images as customers on Ironbank when they spin up their infrastructure.  (Diagram attached of what that image makeup looks like)

![image](https://user-images.githubusercontent.com/71724289/108984713-d737fb00-7655-11eb-9aea-01fceb96a278.png)
